Modules/hpe3par_remote_copy.py

```diff
@@ -0,0 +1,1442 @@
+    return (True, True, "Remote copy group %s stopped successfully." % remote_copy_group_name, {})
+
+def synchronize_remote_copy_group(
+            client_obj,
+            storage_system_username,
+            storage_system_password,
+            remote_copy_group_name,
+            no_resync_snapshot,
+            target_name,
+            full_sync
+            ):
+    if remote_copy_group_name is None:
+        return (False, False, "Synchronize Remote Copy Group failed. Remote Copy Group name is null", {})
+    if len(remote_copy_group_name) < 1 or len(remote_copy_group_name) > 31:
+        return (False, False, "Synchronize Remote Copy Group failed. Remote Copy Group name must be atleast 1 character and not more than 31 characters", {})
+    try:
+        client_obj.login(storage_system_username, storage_system_password)
+        if client_obj.remoteCopyGroupExists(remote_copy_group_name):
+            optional = {
+                'noResyncSnapshot': no_resync_snapshot,
+                'targetName': target_name,
+                'fullSync': full_sync
+            }
+            task = client_obj.synchronizeRemoteCopyGroup(remote_copy_group_name, optional)
+        else:
+            return (False, False, "Remote Copy Group not present", {})
+    except Exception as e:
+        return (False, False, "Synchronize Remote Copy Group failed | %s" % (e), {})
+    finally:
+        client_obj.logout()
+    return (True, True, "Remote copy group %s resynchronize started successfully." % remote_copy_group_name, {})
+
+def delete_remote_copy_group(
+            client_obj,
+            storage_system_username,
+            storage_system_password,
+            remote_copy_group_name,
+            keep_snap
+            ):
+    if storage_system_username is None or storage_system_password is None:
+        return (
+            False,
+            False,
+            "Remote Copy Group delete failed. Storage system username or password is null",
+            {})
+    if remote_copy_group_name is None:
+        return (False, False, "Remote Copy Group delete failed. Remote Copy Group name is null", {})
+    if len(remote_copy_group_name) < 1 or len(remote_copy_group_name) > 31:
+        return (False, False, "Remote Copy Group delete failed. Remote Copy Group name must be atleast 1 character and not more than 31 characters", {})
+    try:
+        client_obj.login(storage_system_username, storage_system_password)
+        if client_obj.remoteCopyGroupExists(remote_copy_group_name):
+            client_obj.removeRemoteCopyGroup(remote_copy_group_name, keep_snap)
+        else:
+            return (True, False, "Remote Copy Group is already present", {})
+    except Exception as e:
+        return (False, False, "Remote Copy Group delete failed | %s" % (e), {})
+    finally:
+        client_obj.logout()
+    return (True, True, "Deleted Remote Copy Group %s successfully." % remote_copy_group_name, {})
+
+def recover_remote_copy_group(
```

Comment:
I guess this is not tested. We can keep comment out this method and its call for now

Reply:
No need to comment function definition since I have commented recover_remote_copy_group function call